labs/src/main/scala/org/scalalabs/basic/lab01/HelloWorldExercise.scala

```diff
@@ -99,11 +99,11 @@ object HelloWorldWithTraits extends HelloTrait with WorldTrait {
  * - combine the 'helloMethod' of HelloTrait and the 'worldMethod' of WorldTrait to create a new message
  * - just replacing the FixMe string would of course be cheating :)
  */
-	def hello:String = "FixMe"
+	def hello:String = helloMethod + " " + worldMethod
```

Comment:
You could also have used string interpolation here like  s"$helloMethod $worldMethod". I'm not sure which one is more Scala-esque, but both work! I actually went back to change my answer when I learned more about string interpolation for the currency part.